### PR TITLE
i18n: Add some missing strings to Polish translation

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

**Review by Gargron — APPROVED** [review]

File: config/locales/pl.yml [hunk 1]
```diff
@@ -50,6 +50,8 @@ pl:
     roles:
       admin: Administrator
       moderator: Moderator
+      staff: Ekipa
```
Comment (ykzts) [inline config/locales/pl.yml]: unused.

File: config/locales/pl.yml [hunk 2]
```diff
@@ -50,6 +50,8 @@ pl:
...
+      user: Użytkownik
```
Comment (ykzts) [inline config/locales/pl.yml]: unused.

File: config/locales/pl.yml [hunk 3]
```diff
@@ -610,8 +613,9 @@ pl:
       limit: Przekroczyłeś maksymalną liczbę przypiętych wpisów
       ownership: Nie możesz przypiąć cudzego wpisu
       private: Nie możesz przypiąć niepublicznego wpisu
-      reblog: Nie możesz przypiąć podbicia wpisu
+      reblog: Nie możesz przypiąć podbicia wpisu/
```
Comment (ykzts) [inline config/locales/pl.yml]: trailing slash is necessary?

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by m4sk1n** [comment]
So now it’s only one string, I'm not sure if there’s need to merge it…

**Review by ykzts — APPROVED** [review]

**Pull request merged**